### PR TITLE
Create professional-development.md

### PR DESCRIPTION
**Pull request by jasonblais**
Moving resource from Google Docs to handbook.

Source: https://docs.google.com/document/d/1sGsbbz1ir3XOGsf3FDySt5P1BpWW6KDqh1qBk1tmAdA/edit#

### CODE REVIEW

**Comment by jasonblais** [comment]
@wiersgallak @justinegeffen friendly reminder on the PR review :)

**Comment by wiersgallak** [comment]
Hi @jasonblais Just curious what prompted adding this to the handbook?

**Comment by jasonblais** [comment]
@wiersgallak I've received questions about prof development tracks in the PM team, so I thought to add the information to the handbook

File: operations/research-and-development/product/professional-development.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Professional Development Tracks
+
+This page outlines professional development tracks available to all product team members (Product Managers, Product Designers, Technical Writers, and Product Analytics Engineers)
```
Comment (wiersgallak) [inline operations/research-and-development/product/professional-development.md]:
```suggestion
This page outlines professional development tracks available to all product team members (Product Managers, Product Designers, Technical Writers, and Product Analytics Engineers). 

The courses listed below are suggestions and not all have been validated.  Some topics do not currently have a suggested course yet. If you take a class that is useful, please consider adding it to this list. 
```

**Comment by wiersgallak** [comment]
@jasonblais Thanks for clarifying.  The original doc was not fully finished, so I just added a note into the first section explaining it a bit.

**Review by justinegeffen — DISMISSED** [review]

**Comment by jasonblais** [comment]
Thanks @wiersgallak, where did you add the note to? Sorry I couldn't find it in the GitHub PR or the doc,

**Review by wiersgallak — CHANGES_REQUESTED** [review]
Seems I forgot to submit these last time.  Sorry for the delay.

**Review by justinegeffen — DISMISSED** [review]

**Review by wiersgallak — DISMISSED** [review]

**Comment by justinegeffen** [comment]
Need to add TOC entry.

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**